### PR TITLE
Clean up log messages

### PR DESCRIPTION
**Pull request by wxiaoguang**
`log.Xxx("%v")` is not ideal, this PR adds necessary context messages. Remove some unnecessary logs.

### CODE REVIEW

File: routers/private/hook_verification.go [hunk 1]
```diff
@@ -66,7 +66,6 @@ func readAndVerifyCommitsFromShaReader(input io.ReadCloser, repo *git.Repository
 		line := scanner.Text()
 		err := readAndVerifyCommit(line, repo, env)
 		if err != nil {
-			log.Error("%v", err)
```
Comment (delvh) [inline routers/private/hook_verification.go:69]: Why was this statement removed?

File: routers/api/v1/notify/repo.go [hunk 2]
```diff
@@ -201,7 +200,6 @@ func ReadRepoNotifications(ctx *context.APIContext) {
 	if !ctx.FormBool("all") {
 		statuses := ctx.FormStrings("status-types")
 		opts.Status = statusStringsToNotificationStatuses(statuses, []string{"unread"})
-		log.Error("%v", opts.Status)
```
Comment (delvh) [inline routers/api/v1/notify/repo.go:204]: :raised_eyebrow: how did that ever end up in productive code?

**Review by delvh — APPROVED** [review]

File: routers/private/hook_verification.go (hunk 1 above)
Comment (wxiaoguang) [reply routers/private/hook_verification.go:69]: Because it just duplicates.

image

File: routers/api/v1/notify/repo.go (hunk 2 above)
Comment (wxiaoguang) [reply routers/api/v1/notify/repo.go:204]: Strict review: contributors complain about stale.
Loose review: end users complain about bugs.

**Review by lunny — APPROVED** [review]

**Pull request merged**